Sources/Logging/Logging.swift

```diff
@@ -97,12 +97,19 @@ extension Logger {
     ///    - line: The line this log message originates from (there's usually no need to pass it explicitly as it
     ///            defaults to `#line`.
     @inlinable
-    public func log(level: Logger.Level,
-                    _ message: @autoclosure () -> Logger.Message,
-                    metadata: @autoclosure () -> Logger.Metadata? = nil,
-                    file: String = #file, function: String = #function, line: UInt = #line) {
+    public func log(
+        level: Logger.Level,
+        _ message: @autoclosure () -> Logger.Message,
+        metadata: @autoclosure () -> Logger.Metadata? = nil,
+        file: String = #file, function: String = #function, line: UInt = #line
```

Comment:
Maybe `file` and `function` should be `StaticString`?

Reply:
@kirilltitov 

- see #4 , `String` in Swift 5 leads to slightly better code and one byte less storage that needs to be copied per call.
- the indentation doesn't match Xcode's default, I think we should use the existing indentation

```swift
func foo(bar: Bar,
         buz: Buz)
```

Reply:
Oh well, I don't like that C-like arguments formatting at all, but whatever :)